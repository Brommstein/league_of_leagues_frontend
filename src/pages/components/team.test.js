import React from 'react';
import Team from './team';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe('Team Component', () => {
    const props = {
        allTeams: {
            teamname: 'Component Test',
            teamabr: 'CTT',
            captainid: 56,
            captain: 'D1',
            topid: 55,
            top: 'D0',
            jungleid: 56,
            jungle: 'D1',
            midid: 57,
            mid: 'D2',
            adcid: 58,
            adc: 'D3',
            supportid: 59,
            support: 'D4'
        },
        userStatus: true,
    }
    it('component', () => {
        const teamComponent = shallow(<Team {...props} />);
        expect(teamComponent).toMatchSnapshot();
    });
});