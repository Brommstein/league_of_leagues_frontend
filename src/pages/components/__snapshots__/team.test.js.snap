// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Team Component component 1`] = `ShallowWrapper {}`;
